api4/file.go

```diff
@@ -103,7 +103,7 @@ func uploadFileStream(c *Context, w http.ResponseWriter, r *http.Request) {
 	if !*c.App.Config().FileSettings.EnableFileAttachments {
 		c.Err = model.NewAppError("uploadFileStream",
 			"api.file.attachments.disabled.app_error",
-			nil, "", http.StatusNotImplemented)
+			nil, "", http.StatusBadRequest)
```

Comment:
I feel like this and the two other changes in this file are not actually bad requests, but there's no other method more specific than 400 that means "the current configuration does not allow you to make this request". So yeah, 400 it is.

Reply:
This should be 403, my mistake, I'll change it.